test/docker-e2e/e2e_test.go

```diff
@@ -41,6 +44,59 @@ func (s *CelestiaTestSuite) SetupSuite() {
 	s.client, s.network = tastoradockertypes.DockerSetup(s.T())
 }
 
+// ExecuteNodeCommand executes a command on a chain node with common parameters automatically added.
+// This reduces boilerplate in tests by setting common flags like --home, --node, --fees, etc.
+func (s *CelestiaTestSuite) ExecuteNodeCommand(ctx context.Context, chainNode tastoratypes.ChainNode, cmd ...string) (string, string, error) {
+	s.Require().Greater(len(cmd), 0, "command must not be empty")
+
+	var finalCmd []string
+	isTxCommand := len(cmd) > 0 && cmd[0] == "tx"
+	isKeysCommand := len(cmd) > 0 && cmd[0] == "keys"
```

Comment:
no need for len check as it is already asserted

Reply:
also this breaks if `celestia-appd` is passed in as part of the command. While not super robust, we could probably just do `contains(cmd, "tx")` rather than explicitly checking an index.